composer.json

```diff
@@ -52,6 +52,7 @@
         "matthiasnoback/symfony-dependency-injection-test": "^4.1",
         "phpstan/phpstan": "^0.12.29",
         "phpstan/phpstan-symfony": "^0.12.10",
+        "phpunit/phpunit": "^9.4",
```

Comment:
Should not be needed too.

Reply:
Then I'd need to add the undefined class line in the `psalm-baseline.xml` file, also `SYMFONY_PHPUNIT_VERSION` is set to `9.4` in `phpunit.xml.dist` and the tests will fail below `9.3.1` due to incompatibility so it makes more sense to add this as dev dependency as well no? What do you think?

Reply:
Then maybe it's better to re-add this to the psalm-baseline (?)

Because it hides a bigger issue.
We're exposing a class in `src/Test` which requires `phpunit/phpunit` without explicitely requiring it.

It should theorically be in the require section, not require-dev then.

But @sonata-project/contributors can see this differently.

Reply:
I see what you mean. Shouldn't the `BlockServiceTestCase` be moved to the` \Tests` namespace then as it's only used in tests? 🤔

Reply:
> I see what you mean. Shouldn't the `BlockServiceTestCase` be moved to the` \Tests` namespace then as it's only used in tests? 🤔

I think it's not like this because the `test` directory is not exported when you install this bundle ; because of https://github.com/sonata-project/SonataBlockBundle/blob/3.x/.gitattributes#L7

Reply:
What about adding `@psalm-suppress UndefinedClass` to the `BlockServiceTestCase` so it's (more) clear this class should be ignored? 

All the other errors are fixed so no `psalm-baseline.xml` should then be required.

Reply:
Could be a good temporary solution.

Reply:
The only usage I can find is in tests so I'm still a bit confused why we should not move it into the `\Tests` namespace but I'm probably missing something.

Running `make test` seems to be working with this change.

Reply:
It's used by others bundle: https://github.com/search?q=org%3Asonata-project+BlockServiceTestCase&type=code